pkg/lib/accountmanagement/service.go

```diff
@@ -184,3 +277,220 @@ func (s *Service) FinishAdding(input *FinishAddingInput) (*FinishAddingOutput, e
 
 	return &FinishAddingOutput{}, nil
 }
+
+type ResendOTPCodeInput struct {
+	UserID       string
+	Channel      model.AuthenticatorOOBChannel
+	LoginID      string
+	isSwitchPage bool
```

Comment:
This is a private field, and I think input should not have private fields. Anything wrong here?